src/main/java/core/AbstractGameState.java

```diff
@@ -394,19 +392,26 @@ public final Stack<IExtendedSequence> getActionsInProgress() {
      * @param playerId - the player observed
      * @return null by default - meaning no tiebreak set for the game; if overwriting, should return the player's tiebreak score
      */
-    public Double getTiebreak(int playerId) {
+    public double getTiebreak(int playerId) {
         return getTiebreak(playerId, 1);
     }
 
     /**
      * @param playerId - the player observed
      * @param tier - if multiple tiebreaks available in the game, this parameter can be used to specify what each one does, applied in the order 1,2,3 ...
-     * @return null - meaning no tiebreak set for the game; if overwriting, should return the player's tiebreak score, given tier
+     * @return Double.MAX_VALUE - meaning no tiebreak set for the game; if overwriting, should return the player's tiebreak score, given tier
      */
-    public Double getTiebreak(int playerId, int tier) {
-        return null;
+    public double getTiebreak(int playerId, int tier) {
+        return Double.MAX_VALUE;
     }
 
+    /**
+     * This sets the number of tieBreak levels in a game.
+     * If we reach this level then we stop recursing.
+     * @return
+     */
+    public int getTiebreakLevels() {return 5;}
```

Comment:
Should this be 5 by default?

Reply:
Five was the random number we mentioned when discussing this :-)

It should be more than enough for most purposes (so no need to override this method in new games); and won't have a performance impact if not implemented.